### PR TITLE
Adds Heartbeat framework and creates group

### PR DESCRIPTION
**Pull request by ladanazita**
Since the Heartbeat logic will be pretty massive, I've created a group called Heartbeats which will contain the video tracking and heartbeat mapping.

This PR also includes the Adobe Heartbeat framework, which is not available in Cocoapods just yet and has to be manually included.

I just want to be sure this file structure makes sense, and since Adobe isn't on Cocoapods just yet, I've included the Framework in Pod/ so users will also be downloading the Heartbeat framework when adding Segment-Adobe-Analytics to their app

### CODE REVIEW

File: Pod/libs/Headers/ADBMediaHeartbeat.h [hunk 1]
```diff
@@ -0,0 +1,333 @@
+/*************************************************************************
+ *
+ * ADOBE CONFIDENTIAL
```
Comment (tonyxiao) [inline Pod/libs/Headers/ADBMediaHeartbeat.h]: @ladanazita should this be included in a public repo?

File: Pod/Classes/Heartbeat/SEGAdobeHeartbeat.m [hunk 2]
```diff
@@ -0,0 +1,8 @@
+//
+//  SEGAdobeHeartbeat.m
+//  AdobeMobileSDK
+//
+//  Created by ladan nasserian on 12/2/17.
+//
+
+#import <Foundation/Foundation.h>
```
Comment (tonyxiao) [inline Pod/Classes/Heartbeat/SEGAdobeHeartbeat.m]: Placeholder for what's to come?

File: Pod/libs/Headers/ADBMediaHeartbeat.h (hunk 1 above)
Comment (ladanazita) [reply Pod/libs/Headers/ADBMediaHeartbeat.h]: They are eventually going to ship to cocoapods, so I'm not too worried — is it possible to have this still be a part of our integration pod but still keep it private?

File: Pod/Classes/Heartbeat/SEGAdobeHeartbeat.m (hunk 2 above)
Comment (ladanazita) [reply Pod/Classes/Heartbeat/SEGAdobeHeartbeat.m]: Yeah I just created the files

File: Pod/libs/Headers/ADBMediaHeartbeat.h (hunk 1 above)
Comment (tonyxiao) [reply Pod/libs/Headers/ADBMediaHeartbeat.h]: Not really. The header files at least have to be public.

**Comment by ladanazita** [comment]
@tonyxiao what changes are you requesting here

**Pull request merged**